Resources/Private/Templates/Frontend/ContentElement/TouristAttraction.html

```diff
@@ -1,119 +1,163 @@
-<html xmlns:f="http://typo3.org/ns/TYPO3/CMS/Fluid/ViewHelpers"
-    data-namespace-typo3-fluid="true">
+<html xmlns:f="http://typo3.org/ns/TYPO3/CMS/Fluid/ViewHelpers" data-namespace-typo3-fluid="true">
+    <nav aria-label="breadcrumb">
```

Comment:
Those templates should work for all projects in general.
The breadcrumb won't work as `/sehenswuerdigkeiten` might not be a proper url in other project setups.

Not sure how we should solve this.